### PR TITLE
Update NetFramework resources

### PR DESCRIPTION
**Pull request by Evangelink**
Tagging @mavasani and @sharwell

### CODE REVIEW

File: src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx [hunk 1]
```diff
@@ -139,91 +139,91 @@
...
   <data name="SetLocaleForDataTypesDescription" xml:space="preserve">
-    <value>The locale determines culture-specific presentation elements for data, such as formatting that is used for numeric values, currency symbols, and sort order. When you create a DataTable or DataSet, you should explicitly set the locale.</value>
+    <value>The locale determines culture-specific presentation elements for data, such as formatting that is used for numeric values, currency symbols, and sort order. When you create a 'DataTable' or 'DataSet', you should explicitly set the locale.</value>
   </data>
   <data name="SetLocaleForDataTypesMessage" xml:space="preserve">
     <value>Set locale for data types</value>
   </data>
   <data name="DoNotMarkServicedComponentsWithWebMethodTitle" xml:space="preserve">
-    <value>Do not mark serviced components with WebMethod</value>
+    <value>Do not mark serviced components with 'WebMethod'</value>
```
Comment (Evangelink) [inline src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx:148]: I know @sharwell was in favor of using the `[WebMethod]` syntax but if we want to be language agnostic it feels weird.

Comment (Evangelink) [inline src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx:148]: Do we want to use the full name?

Comment (Evangelink) [inline src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx:148]: Do we want to add the `Attribute` suffix?

Comment (mavasani) [reply src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx:148]: If we are not using square brackets around the name, then we should use full name with attribute suffix.

**Comment by Evangelink** [comment]
@mavasani PR ready for a new review.

**Pull request closed without merging**